Week2/homework/map-filter.js

```diff
@@ -2,7 +2,7 @@
 
 function doubleOddNumbers(numbers) {
   // Replace this comment and the next line with your code
-  console.log(numbers);
+  return numbers.filter(num => num % 2 !== 0).map(el => el * 2);
```

Comment:
Hi Shyar - this is fine. One small thing I could suggest is to use proper name like `num` again instead of `el` inside of `map` function.

Reply:
@yash-kapila The problem is I created the new branch earlier at this level ``desktop/js2w1/JavaScript2 `` this was my mistake! I should've gone three levels deeper at this point ``desktop/js2w1/JavaScript2/week2/homework ``  then created a new branch.  Will be fine if I remove ``homework-week2`` branch? Then create a new one and change what you have suggested then push everything again?